### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by PavithranGovindh** [HemantKArya/BloomeeTunes#95] [Ideas]
![Screenshot_2024-12-26-17-17-17-828_ls.bloomee.musicplayer.jpg](https://github.com/user-attachments/assets/87a5fca2-a222-4420-937b-a4206dc64f51)
1. Able to download the whole album songs at once

![Screenshot_2024-12-26-17-12-53-280_ls.bloomee.musicplayer.jpg](https://github.com/user-attachments/assets/23c034c4-2b22-4810-926f-ab0c2f413b4d)
2. we played a song and want to hear the whole album (view the whole album)

**Comment by HemantKArya** [comment]
I think both are issues already opened for this app - 
1. https://github.com/HemantKArya/BloomeeTunes/issues/36
2. https://github.com/HemantKArya/BloomeeTunes/issues/88

So, I think we don't need to leave open this topic as a discussion. We will track them there.
btw thanks for suggesting the ideas it will surely help this app. 😊
